tests/conftest.py

```diff
@@ -20,7 +21,88 @@ def pheno_path(datadir):
 
 
 @pytest.fixture(scope='session')
-def pheno(pheno_path):
-    import pharmpy
+def pheno(load_model, pheno_path):
+    return load_model(pheno_path)
 
-    return pharmpy.Model.create_model(pheno_path)
+
+@pytest.fixture(scope='session')
+def load_model(tmp_path_factory):
+
```

Comment:
Since `load_model` has such a similar name to `read_model` and `load_example_model`, I'm thinking that it might make it a little confusing since it sounds like it's a regular `modeling`-function. Perhaps it would be clearer to have it named something like: `load_cached_model` or `load_fixture_model`?

Reply:
Same goes for `load_example` and `create_model`

Reply:
> I was also wondering: how much faster was it with fixtures with copy?

I do not understand?!

Reply:
Yes, I will think about better names!

Reply:
I meant how much did this speed up?

Reply:
On my machine:
  - `tox -e py39 -- pytest -n4 --asyncio-mode=auto --cov --cov-report=term-missing --cov-report xml:.coverage.xml -vv tests`
    - 9m21s  at 43dcfa94 
    - 8m15s with #1035.
  - `tox -e py39 -- pytest -n8 tests/`
    - 4m47  at 43dcfa94 
    - 4m24 with #1035.
  
On GHA the tests consume a total of ~3.5 hours instead of ~4.25 hours.

Reply:
Names have been changed:
  - `load_model` to `load_model_for_test`
  - `load_example` to `load_example_model_for_test`
  - `create_model` to `create_model_for_test_with_pheno_dataset`